readme.md

```diff
@@ -276,10 +276,10 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
   | Package | GitHub Branch | LTS Tag | LTS Start Date | Maintenance End Date |
   | :-----: | :-----------: | :-----: | :------------: | :------------------: |
+  | vcpkg: 2023-07-27 | lts_07_2023 | LTS_07_2023_Ref01 | 2023-07-27 | 2024-07-27 |
```

Comment:
I think the date should be changed last: it's premature to state the release will happen today. Instead, I would first create the release, then check-in the change.
Also, changes should always be made in main then backported. The LTS branch should not contain additional check-ins to those in main unless there is a revision.